unbound-generics.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:       >=1.10
 name:                unbound-generics
-version:             0.4.3
+version:             0.4.3.1
```

```suggestion
version:             0.4.3
```
let's not bump the version as part of this PR, yet.